cluster-sync/sync.sh

```diff
@@ -43,8 +43,6 @@ metadata:
   name: hostpath-provisioner
 spec:
   imagePullPolicy: Always
-  imageRegistry: quay.io/kubevirt
```

Comment:
so we don't need this anymore? how come?

Reply:
The new format of the CRD doesn't have those fields at all. So the webhook is rejecting this completely.

Reply:
I suspect that because I updated k8s to a newer version something is enforcing it now where it wasn't before.

Reply:
oh right yeah, seen that somewhere else too, should really dig out which k8s feature this is